extensions/fastdom-promised.d.ts

```diff
@@ -0,0 +1,8 @@
+declare namespace FastdomPromised {
+  export function clear<T extends Promise<any>>(task: T): void;
+  export function initialize(): void;
```

Comment:
Should this be public?

Reply:
I think so because the extension is exposing the `initialize` method to fastdom. If the `initialize` method was really private then fastdom would not be able to access it.

It seems the `initialize` method is acting like a constructor. In typescript if a constructor is private then it can not be instantiated from outside the class.